.github/workflows/doc-build.yml

```diff
@@ -26,7 +26,7 @@ on:
 
 env:
   # NOTE: west docstrings will be extracted from the version listed here
-  WEST_VERSION: 0.11.0
+  WEST_VERSION: 0.11.1a1
```

Comment:
It's fine to merge with this as-is. There are no doc-build visible changes from a1 to the final release. I'd like to get this merged without having to wait for CI again and will update it tomorrow after the docs include the 0.11 notes, so I can send the release email to the users.

Reply:
I don't have permissions to merge manually yet. @galak - can we get another review so I can merge?